bumble/hci.py

```diff
@@ -2694,17 +2709,45 @@ class HCI_Reset_Command(HCI_Command):
 
 # -----------------------------------------------------------------------------
 @HCI_Command.command(
-    [
+    fields=[
         ('filter_type', 1),
-        ('filter_condition', '*'),
-    ]
+        # TODO: separate this field into condition_type and condition fields
+        # following the spec
+        ('filter_condition_type_and_condition', '*'),
+    ],
+    return_parameters_fields=[('status', STATUS_SPEC)],
 )
 class HCI_Set_Event_Filter_Command(HCI_Command):
     '''
     See Bluetooth spec @ 7.3.3 Set Event Filter Command
     '''
 
 
+# -----------------------------------------------------------------------------
+@HCI_Command.command(
+    return_parameters_fields=[('status', STATUS_SPEC), ('pin_type', PIN_TYPE_SPEC)],
+)
+class HCI_Read_PIN_Type_Command(HCI_Command):
+    '''
+    See Bluetooth spec @ 7.3.5 Read PIN Type Command
+    '''
+
+
+# -----------------------------------------------------------------------------
+@HCI_Command.command(
+    fields=[
+        ('pin_type', 1),
```

Comment:
This should probably use `PIN_TYPE_SPEC` here too.

Reply:
Updated